src/StudentPackage/TakingInputFromUser.java

```diff
@@ -0,0 +1,111 @@
+package StudentPackage;
+
+import java.io.IOException;
+import java.util.*;
+import java.util.Comparator;
+public class TakingInputFromUser {
+
+    public static void TakingInput() throws IOException {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter the no. of Student to enter the details");
+        int noOfStudents = sc.nextInt();
+        int rollNo = 0, ageOfStudent = 0;
+        String addressOfStudent = "", nameOfStudent = "";
+        Queue<String> setOfCourses = new LinkedList<>();
```

Comment:
Why do you need a Queue or a LinkedList ?

Reply:
Done